<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inspector_DockWidget</class>
 <widget class="QDockWidget" name="Inspector_DockWidget">
  <property name="windowTitle">
   <string>Inspector</string>
  </property>
  <widget class="QWidget" name="Inspector_dockWidgetContents">
   <layout class="QGridLayout" name="Inspector_dockWidgetContents_gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="Preview_groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="Preview_groupBox_gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="Image_label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="Options_groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="Options_groupBox_gridLayout">
       <item row="1" column="3">
        <widget class="QComboBox" name="__comboBox">
         <property name="statusTip">
          <string>Map Type Combo Box: Gps Map Display Type.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="Options_horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="Previous_Ibl_Set_pushButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>32</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Previous Ibl Set Push Button: Navigate To Previous Ibl Set In The Current Selected Collection.</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="Next_Ibl_Set_pushButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>32</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Next Ibl Set Push Button: Navigate To Next Ibl Set In The Current Selected Collection.</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
